src/Http/Controllers/SearchController.php

```diff
@@ -101,8 +101,8 @@ public function __invoke(Request $request, $model)
             $query->whereTime($request->get('whereTime'));
         }
 
-        if ($request->has('fields')) {
-            $query->select($request->get('fields'));
+        if ($request->has('select')) {
+            $query->select($request->get('select'));
```

Comment:
why change 'fields' to 'select' this may break a lot of searches

Reply:
shouldnt break anything